doozerlib/cli/release_gen_payload.py

```diff
@@ -88,11 +92,14 @@ def release_gen_payload(runtime, is_name, is_namespace, organization, repository
     brew_session = runtime.build_retrying_koji_client()
     base_target = SyncTarget(  # where we will mirror and record the tags
         orgrepo=f"{organization}/{repository}",
-        istream_name=is_name if is_name else default_is_base_name(runtime.get_minor_version()),
+        istream_name=is_name if is_name else default_is_base_name(runtime),
         istream_namespace=is_namespace if is_namespace else default_is_base_namespace()
     )
 
-    gen = PayloadGenerator(runtime, brew_session, base_target, exclude_arch)
+    if runtime.assembly != 'stream' and 'art-latest' in base_target.istream_name:
```

Comment:
Needs to check if assemblies are enabled here:

```suggestion
    if runtime.assembly and runtime.assembly != 'stream' and 'art-latest' in base_target.istream_name:
```